src/vic-var.h

```diff
@@ -0,0 +1,13 @@
+#ifndef _VIC_VAR_H
+#define _VIC_VAR_H
+
+#include "vic.h"
+
+struct vic_var_t {
+    char name[VIC_VAR_NAME_LEN + 1];
+    char *p_value;
```

Comment:
@vargac just so that we are on the same page, p_value means pointer value here, right?